implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -68,33 +70,32 @@ pub struct CliState {
 }
 
 impl CliState {
-    pub fn try_default() -> Result<Self> {
+    /// Return an initialized CliState
+    /// There should only be one call to this function since it also performs a migration
+    /// of configuration files if necessary
+    pub fn initialize() -> Result<Self> {
```

Comment:
Should we enforce that this function must be called only once?

Reply:
Good idea but how would you do that?

Reply:
That makes sense, thanks!